### ISSUE TITLE
Cannot import/export game data

### ISSUE THREAD

**Issue by somerandomguy24** [flathub/com.github.Anuken.Mindustry#49]
When this is attempted, an empty home folder is shown. This is likely due to the lack of filesystem access permissions.

Steps to reproduce:
1. Open the game 
2. On the main menu, navigate to Settings > Game Data > Import Data
3. Cannot access save file in home folder

**Comment by TheEvilSkeleton** [comment]
You're right. Currently, Mindustry has absolutely no access to your filesystem, other than its own directories. I'm not exactly sure how I would go about allowing permissions, since I don't want to give unnecessary permissions, so I resorted to inconvenience.

I opened https://github.com/Anuken/Mindustry-Suggestions/issues/4247 to ask if the developer can add support to a newer standard - one that will allow importing and exporting data without tinkering with permissions.
